bench/test_flags.py

```diff
@@ -0,0 +1,19 @@
+from bench.utils.factories import FRAME_FACTORIES
```

Comment:
It'd be better to use relative imports here, I think, rather than rely on the local directory being placed in `sys.path`.